app/src/main/java/com/teamagam/gimelgimel/app/network/services/GGMessagingUtils.java

```diff
@@ -60,6 +62,12 @@ public static void sendUserLocationMessageAsync(LocationSample sample) {
     }
 
 
+    public static void sendImageMessageAsync(ImageMetadata meta) {
```

Comment:
This util class is getting really weird. We should find a way to avoid these code dups.